test/assistant-app-test.js

```diff
@@ -28,7 +28,9 @@ const expect = chai.expect;
 const spies = require('chai-spies');
 const { AssistantApp } = require('.././actions-on-google');
 const {
+  apiAiAppRequestBodyNewSessionMock,
```

Comment:
This will probably need to be "dialogflowAppRequestBodyNewSessionMock" in the latest version of the library, if you pull and rebase the latest commit